webxrlayers-1.bs

```diff
@@ -726,8 +728,11 @@ should be inverted.
 
 XRMediaLayerFactory {#xrmedialayerfactorytype}
 -------------------
-The {{XRWebGLLayerFactory}} object is used to create layers that display the content of an {{HTMLVideoElement}}
-.
+The {{XRWebGLLayerFactory}} object is used to create layers that display the content of an {{HTMLVideoElement}}.
```

Comment:
Shouldn't this sentence say refer to `XRMediaLayerFactory` instead of `XRWebGLLayerFactory`?  

`XRMediaLayerFactory` methods do not accept any WebGL things so I was surprised to see WebGL mentioned in the context of it.